packages/fiori-elements-writer/test/__snapshots__/feop.test.ts.snap

```diff
@@ -0,0 +1,3611 @@
+    var ui5Libs = [
+      \\"sap.apf\\",
+      \\"sap.base\\",
+      \\"sap.chart\\",
+      \\"sap.collaboration\\",
+      \\"sap.f\\",
+      \\"sap.fe\\",
+      \\"sap.fileviewer\\",
+      \\"sap.gantt\\",
+      \\"sap.landvisz\\",
+      \\"sap.m\\",
+      \\"sap.ndc\\",
+      \\"sap.ovp\\",
+      \\"sap.rules\\",
+      \\"sap.suite\\",
+      \\"sap.tnt\\",
+      \\"sap.ui\\",
+      \\"sap.uiext\\",
+      \\"sap.ushell\\",
+      \\"sap.uxap\\",
+      \\"sap.viz\\",
+      \\"sap.webanalytics\\",
+      \\"sap.zen\\"
+    ];
+    function getKeys(libOrComp,libOrCompKeysString) {
+      Object.keys(libOrComp).forEach(function (libOrCompKey) {
+        // ignore libs or Components that start with SAPUI5 delivered namespaces
+        if (!ui5Libs.some(function (substring) { return libOrCompKey === substring || libOrCompKey.startsWith(substring + \\".\\"); })) {
+          if (libOrCompKeysString.length > 0) {
+            libOrCompKeysString = libOrCompKeysString + \\",\\" + libOrCompKey;
+          } else {
+            libOrCompKeysString = libOrCompKey;
+          }
+        }
+      });
+      return libOrCompKeysString;
+    }
+    return new Promise(function (resolve, reject) {
+      $.ajax(url)
+        .done(function (manifest) {
+          if (manifest) {
+            if (
+              manifest[\\"sap.ui5\\"] &&
+              manifest[\\"sap.ui5\\"].dependencies
+            ) {
+              if (manifest[\\"sap.ui5\\"].dependencies.libs){
+                result = getKeys(manifest[\\"sap.ui5\\"].dependencies.libs, result)
+              }
+              if (manifest[\\"sap.ui5\\"].dependencies.components){
+                result = getKeys(manifest[\\"sap.ui5\\"].dependencies.components, result)
+              }
+            }
+            if (
+              manifest[\\"sap.ui5\\"] &&
+              manifest[\\"sap.ui5\\"].componentUsages
+            ) {
+              result = getKeys(manifest[\\"sap.ui5\\"].componentUsages, result)
+            }
+          }
+          resolve(result);
+        })
+        .fail(function (error) {
+          reject(new Error(\\"Could not fetch manifest at '\\" + manifestPath));
+        });
+    });
+  };
+  /**
+   * Registers the module paths for dependencies of the given component.
+   * @param {string} manifestPath The the path to the app manifest path
+   * for which the dependencies should be registered.
+   * @returns {Promise} A promise which is resolved when the ajax request for
+   * the app-index was successful and the module paths were registered.
+   */
+  sap.registerComponentDependencyPaths = function (manifestPath) {
+    /*eslint-disable semi, consistent-return*/
+    return fioriToolsGetManifestLibs(manifestPath).then(function (libs) {
+      if (libs && libs.length > 0) {
+        var url = \\"/sap/bc/ui2/app_index/ui5_app_info?id=\\" + libs;
+        var sapClient = jQuery.sap.getUriParameters().get(\\"sap-client\\");
```

Comment:
we should not use jQuery.sap in the code we deliver as this has been deprecated since a lot of versions by. ui5

Reply:
Im using a provided file by @devinea - maybe he can comment.